src/electronRendererEnhancer.js

```diff
@@ -73,10 +73,10 @@ export default function electronRendererEnhancer({
 
       // Dispatches from other processes are forwarded using this ipc message
       ipcRenderer.on(`${globalName}-browser-dispatch`, (event, { action, sourceClientId }) => {
-        action = JSON.parse(action);
+        const actionParsed = JSON.parse(action);
```

Comment:
Can you double check this line.

I changed it because of a linting error "assignment to function parameter 'action'" but I've just had some eerie déjà vu that I did this before in a previous PR and it caused a bug.